frontend/public/components/dashboard/project-dashboard/project-dashboard.tsx

```diff
@@ -49,10 +45,11 @@ export const getNamespaceDashboardConsoleLinks = (
   });
 };
 
-const ProjectDashboard_: React.FC<ProjectDashboardReduxProps & ProjectDashboardProps> = ({
-  obj,
-  consoleLinks,
-}) => {
+export const ProjectDashboard: React.FC<ProjectDashboardProps> = ({ obj }) => {
+  const [consoleLinks] = useK8sWatchResource<K8sResourceKind[]>({
+    isList: true,
+    kind: referenceForModel(ConsoleLinkModel),
```

Comment:
I think we probably want `optional: true` each place we watch these links.

Reply:
D'oh.  Thanks.  I got distracted trying to figure out how to avoid the copy/pasting of the resource and completely forgot to add it.  Fixed.